public/app/plugins/panel/geomap/utils/location.ts

```diff
@@ -142,6 +188,27 @@ export function getLocationFields(frame: DataFrame, location: LocationFieldMatch
   return fields;
 }
 
+function getLocationJson(lookupSrc: LookupSource): any {
+  // built-in file
```

Comment:
use:
```
getBackendSrv().get( path )
```

for the 'file' type we can just make sure it will work with relative paths '/' stats in the public folder, and if the file has 'http://..' an absolute path will also work